### PR TITLE
I have added message of how to quit and a quit function

### LINKED ISSUE #4: Quitting the game
The user should be able to quit with typing the `quit`.

### PR DESCRIPTION
**Pull request by RhysDaBoi**
Issue resolved: https://github.com/Shildus/rsp_game_demo/issues/4
The quit function also works with upper or lowercase letters.

### CODE REVIEW

File: game.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
 
 print("Welcome to our rock paper scissors game!")
+print("If you wish to quit type 'quit' into the console")
 while True:
     player_choice = input("Please enter your choice from rock, paper or scissors. ")
-    print(player_choice)
\ No newline at end of file
+    print(player_choice)
+    if player_choice.lower() == "quit":
```
Comment (Mega-Roman) [inline game.py]: 🐼  have an option where you can simply type "q" to quit

**Review by professionalSkillIssue — APPROVED** [review]
Looks good to me.

**Review by Mega-Roman — CHANGES_REQUESTED** [review]
Works fine but could you please add a final message, something like "thank you for playing our game!"?

**Review by Mega-Roman — APPROVED** [review]
wonderful, thank you, looking good now 😄

**Pull request merged**